src/lib/http-server.ts

```diff
@@ -37,13 +37,13 @@ export class HttpServer {
 
   /**
    * Creates a new instance of HttpServer.
-   * @param {RequestListener} requestListener The function that will handle the server's requests.
```

Comment:
with jsdoc supporting typescript, having the definition in the doco raises a warning

cf. https://github.com/gajus/eslint-plugin-jsdoc/blob/main/docs/rules/no-types.md#no-types